number-of-1-bits/bhyun-kim.py

```diff
@@ -0,0 +1,31 @@
+"""
+191. Number of 1 Bits
+https://leetcode.com/problems/number-of-1-bits/
+
+Solution:
+    - Create a counter
+    - For each bit in the input number:
+        - If the bit is 1:
+            - Increment the counter
+    - Return the counter
+
+Time complexity: O(1)
+    - The while loop runs 32 times
+
+Space complexity: O(1)
+    - No extra space is used
+"""
+
+
+class Solution:
+    def hammingWeight(self, n: int) -> int:
+        output = 0
+
+        i = 1 << 31
+        while i > 0:
+            if (n & i) != 0:
+                output += 1
+
+            i = i // 2
```

Comment:
위에서 왼쪽으로 31칸 시프트를 하셨으니, 여기서는 오른쪽으로 1칸씩 시프트해주시는 게 더 직관적이지 않을까요?

```suggestion
            i = i >> 1
```

Reply:
넵 ㅎㅎ 추천하신대로 통일감 있게 바꿔 봤습니다.